content/docs/started/getting-started-gke.md

```diff
@@ -33,15 +33,18 @@ Create an OAuth client ID to be used to identify Cloud IAP when requesting acces
    * Configure the [consent screen](https://console.cloud.google.com/apis/credentials/consent).
    * Under **Email address**, select the address that you want to display as a public contact. You must use either your email address or a Google Group that you own.
    * In the **Product name** box, enter a suitable name like `kubeflow`.
-   * Under **Authorized domains**, enter
+   * If you see **Authorized domains**, enter
 
      ```
      <project>.cloud.goog
      ```
-
-     where \<project\> is your Google Cloud Platform (GCP) project ID.
-
+     * where \<project\> is your Google Cloud Platform (GCP) project ID.
+     * If you are using your own domain e.g. **acme.com** you should add that as well
+     * Depending on your project **Authorized domains** might not be an option
```

Comment:
Suggested rewording, to avoid sounding like folks are missing out on something:

The **Authorized domains** option appears only for certain project configurations. If you don't see the option, that's fine.